src/app/tab1/tab1.page.html

```diff
@@ -1,31 +1,27 @@
 <ion-header>
   <ion-toolbar>
     <ion-title>
-      Tab One
+      Tv Shows
     </ion-title>
   </ion-toolbar>
 </ion-header>
 
 <ion-content>
   <ion-list lines="none">
     <ion-list-header>
-      <ion-label>Resources</ion-label>
+      <ion-label>trending tv shows</ion-label>
```

Comment:
Hoofdletter gebruiken 
```suggestion
      <ion-label>Trending TV-Shows</ion-label>
```